src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateRes = date.split(fromFormat[3]);
+  const result = [];
+  let year = 0;
+  let month = 0;
+  let day = 0;
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'YY':
+        year = i;
+        break;
+      case 'YYYY':
+        year = i;
+        break;
```

Comment:
Optional

You can move this to a one conditional to avoid duplication of code and don`t repeat yourself
```suggestion
    case 'YY':
 		case 'YYYY':
    	year = i;
   	  break;
   
```